### ISSUE TITLE
Mention installing skpm and sketchtool in Quick Start

### ISSUE THREAD

**Issue by jayliu50** [airbnb/react-sketchapp#34]
_I am..._

| -------------------------------------------------------------------------------------------------
| Reporting a bug or issue
| -------------------------------------------------------------------------------------------------

Getting Started documentation should mention skpm and sketchtool as dependencies to install first. It wasn't apparent to me, that I needed to install these, as I'm a complete n00b in the sketch plugin space.

**Observed behavior:**

**How to reproduce:**
Without having skpm and sketchtool installed, try following the current quickstart instructions.

> First, make sure you have installed Sketch version 43+, & a recent npm.
> 
> Open a new Sketch file, then in a terminal:
> 
> git clone https://github.com/airbnb/react-sketchapp.git
> cd react-sketchapp/examples/basic-setup && npm install
> 
> npm run render
> Next, check out some more examples!

**Comment by mathieudutour** [comment]
skpm is install when you run `npm install` and `sketchtool` is bundled in Sketch.app so there shouldn't be any additional step. What was the error?

**Comment by garrettmaring** [comment]
With Sketch v43.2 you see this if you don't have a Sketch file open.

![screen shot 2017-04-26 at 12 27 39 pm](https://cloud.githubusercontent.com/assets/8270120/25452819/c2e53322-2a7b-11e7-943c-37d627c81b8f.png)

**Comment by jayliu50** [comment]
@mathieudutour 

```
$ skpm build --watch --run
🖨  Copied src/manifest.json in 14ms
Executing post-build scripts
⛏  Built ./main.js in 3218ms
Unknown command ‘run’

Usage: sketchtool <command> [<args>]
                  [--formats=<string> | -f <string>]
                  [--use-id-for-name{=YES|NO} | --no-use-id-for-name | -n {<YES|NO>}]
                  [--max-size=<float> | -m <float>]
                  [--item=<string> | -i <string>]
                  [--items=<string> | -a <string>]
                  [--background=<string> | -g <string>]
                  [--compression=<float> | -c <float>]
                  [--reveal{=YES|NO} | --no-reveal | -r {<YES|NO>}]
                  [--outputJSON=<path> | -J <path>]
                  [--include-symbols{=YES|NO} | --no-include-symbols | - {<YES|NO>}]
                  [--bounds=<rectangle> | -b <rectangle>]
                  [--version | -v] [--filename=<string> | -F <string>]
                  [--scales=<path> | -s <path>]
                  [--compact{=YES|NO} | --no-compact | -k {<YES|NO>}]
                  [--overwriting{=YES|NO} | --no-overwriting | -V {<YES|NO>}]
                  [--trimmed{=YES|NO} | --no-trimmed | -t {<YES|NO>}]
                  [--group-contents-only{=YES|NO} | --no-group-contents-only | -G {<YES|NO>}]
                  [--help | -h]
                  [--include-namespaces{=YES|NO} | --no-include-namespaces | -N {<YES|NO>}]
                  [--progressive{=YES|NO} | --no-progressive | -p {<YES|NO>}]
                  [--save-for-web{=YES|NO} | --no-save-for-web | -w {<YES|NO>}]
                  [--output=<path> | -o <path>]

Commands:
	dump                 Dump out the structure of a document as JSON.

	export artboards     Export one or more artboards
	export layers        Export one or more layers
	export pages         Export an area from one or more pages
	export preview       Export a preview image for a document
	export slices        Export one or more slices

	help                 Show this help message.

	list artboards       List information on the document's artboards.
	list formats         List the supported export formats.
	list layers          List information on all of the document's layers.
	list pages           List information on the document's pages.
	list slices          List information on the document's slices.

	metadata             List the metadata for a document.
	notes                Output the release notes.
	show                 Show the location of the various sketch folders.


See ‘sketchtool help <command>’ for more information on a specific command.

/Users/jql4791/.nvm/versions/node/v7.9.0/lib/node_modules/skpm/node_modules/webpack-shell-plugin/lib/index.js:168
        throw error;
        ^
1
```

**Comment by jongold** [comment]
If you see 'unknown command run' then your version of sketchtool is old.

It sounds like some people have Sketch installed at a different path, which might explain it - see #27 for how to override the Sketch path

**Comment by mathieudutour** [comment]
I published skpm@0.9.4 which will warn you if it doesn't find the `run` command with a nicer message
